spec/lib/saml_idp/incoming_metadata_spec.rb

```diff
@@ -26,6 +26,7 @@ module SamlIdp
     it 'should properly set sign_assertions to false' do
       metadata = SamlIdp::IncomingMetadata.new(metadata_1)
       expect(metadata.sign_assertions).to eq(false)
+      expect(metadata.entity_id).to be_a(String)
```

Comment:
This test seems to be specific to `sign_assertions` - could we move it to it's own `it` case and test that it extracts the actual information we expect rather than simply checking the type?

Reply:
I have moved to external it and validating against actual value. Could you please take a look at it.